recipes/plugins_example_document.ipynb

```diff
@@ -194,12 +191,16 @@
     "\n",
     "            print(\"Asset added to review\")\n",
     "\n",
-    "            \n",
-    "        accuracy = 100 - len(issues_array) / len(annotations_for_job0)*100\n",
+    "        accuracy = (\n",
+    "            100 - len(issues_array) / len(label[\"jsonResponse\"][\"JOB_0\"][\"annotations\"]) * 100\n",
+    "        )\n",
     "\n",
-    "        print(\"Accuracy score computed\")\n",
+    "        print(accuracy)\n",
+    "        self.kili.update_properties_in_assets(\n",
+    "            asset_ids=[asset_id], json_metadatas=[\"{'accuracy': accuracy}\"]\n",
+    "        )\n",
     "\n",
-    "        kili.update_properties_in_assets(asset_ids=[asset_id], json_metadatas=[{'accuracy': accuracy}])"
```

Comment:
where do we use all the billing documents ?

Reply:
it's better to avoid image files in a repo, it is heavy & we can simply use URLs

Reply:
in the notebook !

Reply:
I should put those in a bucket then ? The problem is that the notebook is a form of example to upload asset from your computer as well